index.html

```diff
@@ -31,15 +50,15 @@ <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
                 <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box1">Box 2</div>
+                <div class="box2">Box 3</div>
+                <div class="box3">Box 4</div>
+                <div class="box4">Box 5</div>
+                <div class="box5">Box 6</div>
+                <div class="box6">Box 7</div>
+                <div class="box7">Box 8</div>
+                <div class="box8">Box 9</div>
+                <div class="box9">Box 10</div>
             </div>
```

Comment:
Let's tweak this just a bit. We can add multiple classes to an element simply by putting a space between each class name inside the quotes:

```suggestion
<div class="boxes">
   <div class="box box1">Box 1</div>
   <div class="box box2">Box 2</div>
   <div class="box box3">Box 3</div>
   <div class="box box4">Box 4</div>
   <div class="box box5">Box 5</div>
   <div class="box box6">Box 6</div>
   <div class="box box7">Box 7</div>
   <div class="box box8">Box 8</div>
   <div class="box box9">Box 9</div>
   <div class="box box10">Box 10</div>
</div>
```